### ISSUE TITLE
Hello guys I am trying to receive data from opcua server in javascript and i have this error on terminal of my raspberry pi 

### ISSUE THREAD

**Issue by FabioAlmeida25** [node-opcua/node-opcua#1025]
![Capturar](https://user-images.githubusercontent.com/86373561/125054226-e61bc900-e09d-11eb-89f7-6cb08b728a6e.PNG)
![image](https://user-images.githubusercontent.com/86373561/125054266-f03dc780-e09d-11eb-88b4-add2f11be6ef.png)
![image](https://user-images.githubusercontent.com/86373561/125054320-fc298980-e09d-11eb-8287-952939c7b358.png)
![image](https://user-images.githubusercontent.com/86373561/125054368-077cb500-e09e-11eb-9910-2e93a1361dc7.png)

**Comment by AndreasHeine** [comment]
check your NodeId with UaExpert!
looks like a Siemens PLC here is a screenshot from my Test PLC:
![grafik](https://user-images.githubusercontent.com/56362817/125056644-b7efb680-e0a8-11eb-8a68-71c8f4216c18.png)

**Comment by FabioAlmeida25** [comment]
Hello @AndreasHeine for your help.
In my case don't apear that way and I don't know why.
![image](https://user-images.githubusercontent.com/86373561/125058174-ecac3f80-e0a1-11eb-9f50-27a385b68093.png)

**Comment by AndreasHeine** [comment]
ah ok use -> "ns=4;i=4" in your case your "IdentifierType" is numeric not string!

**Comment by FabioAlmeida25** [comment]
Error again:
![image](https://user-images.githubusercontent.com/86373561/125058891-9ee40700-e0a2-11eb-9049-027378ae5e91.png)
![image](https://user-images.githubusercontent.com/86373561/125058931-a9060580-e0a2-11eb-952f-bf87ddf23631.png)
![image](https://user-images.githubusercontent.com/86373561/125058950-b1f6d700-e0a2-11eb-90fa-1015674f4ba0.png)

**Comment by FabioAlmeida25** [comment]
Ok I receive the value 10, the same value as my variável_1, so its ok. Sorry my english by the way.

**Comment by FabioAlmeida25** [comment]
I haved tried that way, but when I opened my webpag my graph don't shown any value. And I know it's strange because I established connection between my webserver and  the client using python language. And I used in that case (ns=4;i=4). My problem now I think it is in the rest of the code.

![image](https://user-images.githubusercontent.com/86373561/125060292-2120fb00-e0a4-11eb-966b-15a1ce60d3ee.png)
